app/models/user.rb

```diff
@@ -62,4 +52,23 @@ def activate
   def send_activation_email
     UserMailer.account_activation(self).deliver_now
   end
+
+  def create_reset_digest
+    self.reset_token = User.new_token
+    update_attributes reset_digest: User.digest(reset_token), reset_sent_at: Time.zone.now
```

Comment:
change `update_attributes`  to `update_columns`

Reply:
done